.github/workflows/chromatic.yml

```diff
@@ -2,6 +2,8 @@ name: Chromatic PR
 
 on:
   push:
+    branches-ignore:
+      - renovate/**
 
```

Comment:
The renovate PRs eat up our snapshots, while the risk of them breaking the UI is fairly low. We will still run on main pushes anyway.